src/mmw/js/src/app.js

```diff
@@ -261,6 +265,7 @@ function toggleCompareViewForITSIScreenshot(adjustForScreenshot) {
         } else if ($(compareTableRow).length) {
             $(compareTableRow).removeClass(itsiCompareRow);
         }
+        $(compareSections + '> div').css('margin-top', '');
```

Comment:
Out of curiosity I checked what happened if we removed this line (on the notion that it might prevent the post-screenshot `scrollTop` behavior). Removing it did leave the scroll position visibly the same BUT it also truncated the overflow altogether such that I couldn't scroll back above the visible lines.